### DISCUSSION TITLE
Blog comment on Running Large Language Models Privately - privateGPT and Beyond

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#884] [Docs/blog comments]
# Blog comment on Running Large Language Models Privately - privateGPT and Beyond

A discussion on data privacy and privacy-preserving machine learning for LLMs

https://weaviate.io/blog/private-llm?utm_source=newsletter.weaviate.io&utm_medium=newsletter&utm_campaign=local-vector-database-ai-hackathons-and-berlin-buzzwords

**Comment by Tej-Sharma** [comment]
Interesting article, does Weaviate have any support for any of these techniques?
